src/utils/navigate.ts

```diff
@@ -1,7 +1,9 @@
 /** Reloads Filings UI using business id instead of temporary registration number. */
-export const reloadWithBusinessId = () => {
+export const reloadWithBusinessId = (filing: ApiResponseFilingI) => {
   // build the URL to the business dashboard with the business id and any URL parameters
   // const url = getDashboardUrl + filing.businessIdentifier // + this.$route.fullPath
-  const url = window.location.href
-  window.location.assign(url)
+  const businessIdentifier = filing.businessIdentifier
+  const url = new URL(window.location.href)
+  url.pathname = `/${businessIdentifier}`
+  window.location.assign(url.toString())
```

Comment:
How can I test this?

Reply:
waiting for Patricks PR to get merged in